zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookServer.java

```diff
@@ -382,6 +396,7 @@ private void broadcastToNoteBindedInterpreter(String interpreterGroupId,
       for (String id : ids) {
         if (id.equals(interpreterGroupId)) {
           broadcast(note.getId(), m);
+          broadcastToWatchers(note.getId(), StringUtils.EMPTY, m);
```

Comment:
Looks like `broadcastToWatchers()` is called inside of `broadcast(String noteId, Message m)` already.